2.0.2/buster/build-time-pip-constraints.txt

```diff
@@ -1,87 +1,88 @@
 # Constraint the version pip will install during docker build
 
 adal==1.2.7
-alembic==1.5.8
+alembic==1.6.5
 amqp==2.6.1
-apache-airflow-providers-amazon==1!1.3.0
+apache-airflow==1!2.0.2+astro.4
+apache-airflow-providers-amazon==1!1.4.0
 apache-airflow-providers-celery==1!1.0.1
 apache-airflow-providers-cncf-kubernetes==1!1.2.1
 apache-airflow-providers-elasticsearch==1!1.0.4
-apache-airflow-providers-ftp==1!1.0.1
-apache-airflow-providers-google==1!2.2.0
+apache-airflow-providers-ftp==1!1.1.0
+apache-airflow-providers-google==1!3.0.0
```

Comment:
This is a breaking change, bumps a major version

Reply:
Updated with pinned google provider of 2.2.0